cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/WriterOptions.java

```diff
@@ -45,6 +45,7 @@ public enum WriterOptions implements WriterOption
     COMMIT_THREADS_PER_INSTANCE,
     COMMIT_BATCH_SIZE,
     SKIP_EXTENDED_VERIFY,
+    SKIP_ROWS_VIOLATING_CONSTRAINTS,
```

Comment:
Can you maybe add a comment here that this feature is only available in Cassandra 5.0 version? Or is it 6?

Reply:
@frankgh thank you for the review but I am not sure this approach is still viable based on the discussion in the related ticket. Did I interpret that comment correctly when I think that what I did here is itself not enough to consider CASSANALYTICS-32 to be resolved as we need to expose failed rows _somehow_, but this I am trying to do here is a nice to have / first step towards that direction which is valuable already to have as such?

https://issues.apache.org/jira/browse/CASSANALYTICS-32?focusedCommentId=17980384&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17980384

Reply:
Yeah, I saw the comment from Yifan after I reviewed the PR. I think it's a good first approach, you need to explicitly enable this feature in order to write data that does not violate the constraints.